src/tests/ftest/dfuse/sparse_file.py

```diff
@@ -57,8 +56,7 @@ def test_sparsefile(self):
 
         # create large file and perform write to it so that if goes out of
         # space.
-        sparse_file = str(self.dfuse.mount_dir.value + "/" +
-                          "sparsefile.txt")
+        sparse_file = str(self.dfuse.mount_dir.value + "/" + "sparsefile.txt")
```

Comment:
This would be better implemented as:
```suggestion
        sparse_file = os.path.join(self.dfuse.mount_dir.value, "sparsefile.txt")
```

Reply:
Yes I agree.  I was going for the path of least resistance on this PR but I'm happy to re-push it.